scripts/extract_example.py

```diff
@@ -0,0 +1,19 @@
+from source_code import *
```

Comment:
please move testing to the `tests` directory and use `from scripts.source_code import ...`